src/sql/workbench/contrib/assessment/test/browser/asmtActions.test.ts

```diff
@@ -0,0 +1,198 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import * as TypeMoq from 'typemoq';
+import * as assert from 'assert';
+import { IAssessmentComponent } from 'sql/workbench/contrib/assessment/browser/asmtResultsView.component';
+import { AssessmentType, AssessmentTargetType } from 'sql/workbench/contrib/assessment/browser/consts';
+import {
+	AsmtServerInvokeItemsAction,
+	AsmtServerSelectItemsAction,
+	AsmtExportAsScriptAction,
+	AsmtSamplesLinkAction,
+	AsmtDatabaseInvokeItemsAction,
+	AsmtDatabaseSelectItemsAction
+} from 'sql/workbench/contrib/assessment/browser/asmtActions';
+import { AssessmentService } from 'sql/workbench/services/assessment/common/assessmentService';
+import { NullAdsTelemetryService } from 'sql/platform/telemetry/common/adsTelemetryService';
+import { URI } from 'vs/base/common/uri';
+import { IOpenerService } from 'vs/platform/opener/common/opener';
+import { IConnectionManagementService } from 'sql/platform/connection/common/connectionManagement';
+import { TestConnectionManagementService } from 'sql/platform/connection/test/common/testConnectionManagementService';
+import { NullLogService } from 'vs/platform/log/common/log';
+import { ConnectionProfile } from 'sql/platform/connection/common/connectionProfile';
+/**
+ * Class to test Assessment Management Actions
+ */
+class AssessmentTestViewComponent implements IAssessmentComponent {
+	showProgress(mode: AssessmentType) { return undefined; }
+	showInitialResults(result: azdata.AssessmentResult, method: AssessmentType) { return undefined; }
+	appendResults(result: azdata.AssessmentResult, method: AssessmentType) { }
+	stopProgress(mode: AssessmentType) { return undefined; }
+	resultItems: azdata.AssessmentResultItem[];
+	isActive: boolean = true;
+}
+
+class OpenerServiceStub implements IOpenerService {
```

Comment:
Put this in a more generic place so other tests can easily access it (I'd put it in the sql folders though to minimize changes/conflicts in the vscode side)

Reply:
I have placed it to the sql/platform/opener/common